server/cmd/mmctl/printer/printer.go

```diff
@@ -33,7 +33,7 @@ type Printer struct { //nolint
 	pager         bool
 	Quiet         bool
 	Lines         []interface{}
-	ErrorLines    []interface{}
+	ErrorLines    []string
```

Comment:
Why making this change? For consistency I'd try to keep the same type for both `Lines` and `ErrorLines`

Reply:
I made the change to allow the test to check `ErrorLines` more easily. 

While `ErrorLines` can only contain `string`, `Lines` does contain arbitrary information. I don't think it is worth keeping consistency between them.